app/services/chasers/candidate.rb

```diff
@@ -0,0 +1,22 @@
+module Chasers
+  module Candidate
+    # chaser: [days_ago, days_ago]
+    OFFER_CHASERS_TO_INTERVALS = {
+      offer_10_day: [20, 10],
+      offer_20_day: [30, 20],
+      offer_30_day: [40, 30],
+      offer_40_day: [50, 40],
+      offer_50_day: [60, 50],
+    }.freeze
+
+    def self.chaser_types
```

Comment:
I think this is also used only for a worker spec. Can't we have `Chasers::Candidate::OFFER_CHASERS_TO_INTERVALS.keys` instead in `offer_worker_spec`?

Reply:
I think using public value constants like this should be avoided. I would like to see all constants accessed through public methods. Littering the code with value constants blurs the lines between public and private interfaces.

Reply:
I get that, but for a spec? You avoid sharing a constant because when it gets referenced in other classes, you can't change its structure without changing other code. But we can change specs freely. Or we'd `private_constant :OFFER_CHASERS_TO_INTERVALS` it. You can keep it as it is, but similar to the block issue, having public code that only gets called in specs can be confusing. For example, in production, that method will never be called.